packages/Card/src/components/Header/Header.js

```diff
@@ -1,27 +1,47 @@
 import React from "react";
 import PropTypes from "prop-types";
-
+import Button from "@paprika/button";
 import * as sc from "./Header.styles";
 
 const propTypes = {
   /** Primary content. */
   children: PropTypes.node,
+
+  /** Callback to be executed when the button is clicked or activated. */
+  onButtonClick: PropTypes.func,
+
+  /** Render an icon. */
+  icon: PropTypes.node,
 };
 
 const defaultProps = {
   children: null,
+  onButtonClick: () => {},

```

Comment:
This should be `null` by default, otherwise you're always going to render a `<Button.Icon>` below.